zerver/lib/events.py

```diff
@@ -1523,6 +1515,14 @@ def _draft_update_action(i: int) -> None:
             state["user_topics"].append({x: event[x] for x in fields})
     elif event["type"] == "has_zoom_token":
         state["has_zoom_token"] = event["value"]
+    elif event["type"] == "reload_web_client":
+        # This is an unlikely race, where the queue was created with a
+        # previous Tornado process, which restarted, and subsequently
+        # was told by restart-server to tell its old clients to
+        # reload.  We warn, since we do not expect this race to be
+        # possible, but the worst expected outcome is that the client
+        # retains the old JS instead of reloading.
+        logging.warning("reload_web_client during apply_events")
```

Comment:
I guess this is the cause of the test failure; fix should be to just have the test code capture the log line.